html/pages/index.html

```diff
@@ -1,4 +1,198 @@
 ---
 layout: default
-title: Home
+title: Movies&Stuff
 ---
+
+<nav class="navbar">
+  <h1 class="logo">
+    <a href="#">Moves & Stuff</a>
+  </h1>
+
+  <ul class="menu">
+    <li><a href="#movies">Movies</a></li>
+    <li><a href="#tv-shows">TV Shows</a></li>
+    <li><a href="#">Contacts</a></li>
+  </ul>
+</nav>
+
+<header class="poster">
+  <article class="featured-movie">
+    <figure class="featured-movie-poster">
+      <img src="{{root}}/images/deadpool-preview.jpg"
+        alt="new movie Deadpool">
+    </figure>
+
+    <section class="featured-movie-information">
+      <h2 class="featured-movie-name">Deadpool</h2>
+
+        <section class="featured-muvie-rating">
+          <img src="{{root}}/images/star.png"
+               class="featured-muvie-rating-item">
+          <img src="{{root}}/images/star.png"
+               class="featured-muvie-rating-item">
+          <img src="{{root}}/images/star.png"
+               class="featured-muvie-rating-item">
+          <img src="{{root}}/images/star.png"
+              class="featured-muvie-rating-item">
+          <img src="{{root}}/images/star-copy.png"
+              class="featured-muvie-rating-item">
+        </section>
+
+      <p class="featured-movie-description">
+        A fromer Special Forces operative turned mercenary is subjeced to rouge
+        experiment that leaves him with accelerated healing powers, adopting
+        the alter ego Deadpool
+      </p>
+
+      <footer class="featured-movie-meta">
+        R | 180 min | Action,Adventure,Comedy |
+        <time datetime="12-02-2016"> 12 February 2016</time>
+      </footer>
+    </section>
+  </article>
+</header>
+
+<section class="popular-movies" id="movies">
+  <a href="#">
+    <h3>Popular Movies</h3>
```

Comment:
`h1` or `h2`. Don't look on the physical size of headings in pixels, look on its position in the document's hierarchy.